lib/gems/pending/VMwareWebService/MiqVimDataStore.rb

```diff
@@ -287,7 +286,7 @@ def get_file_content(filepath)
     /(\d*)\.(\d*)/ =~ @invObj.about['version']
     raise "get_file_content not supported on [#{@invObj.about['fullName']}]" if $1.to_i < 3 || ($1.to_i == 3 && $2.to_i < 5)
 
-    fileUrl = "https://#{@invObj.server}/folder/#{MIQEncode.base64Encode(filepath)}?dsName=#{MIQEncode.base64Encode(@name)}"
+    fileUrl = "https://#{@invObj.server}/folder/#{Base64.encode64(filepath)}?dsName=#{Base64.encode64(@name)}"
```

Comment:
I'm :+1: for dropping the dependency, but...
```ruby
MIQEncode.base64Decode(Base64.encode64("abc"))
=> "YWJj\n"
Base64.decode64(MIQEncode.base64Encode("abc"))
=> "i\xB7"
```

Apparently this is using URI.encode and decode.  Thanks for finding this, I probably would have hit it in the future.

I think you want
```ruby
URI.encode(data, /[^#{URI::PATTERN::UNRESERVED}]/)
```